test/e2e/multicast_test.go

```diff
@@ -68,10 +71,42 @@ func TestMulticast(t *testing.T) {
 	}
 	// Only check receiver router in noEncap mode.
 	checkReceiverRoute := encapMode == config.TrafficEncapModeNoEncap
-	runMulticastTestCases(t, data, nodeMulticastInterfaces, checkReceiverRoute)
+	runMulticastTestCases(t, data, nodeMulticastInterfaces, data.testNamespace, checkReceiverRoute, true)
+
+	t.Run("testMulticastWithFlexibleIPAM", func(t *testing.T) {
```

Comment:
Question: do we need to revert antrea configurations after this case is completed? My concern is if the subsequent cases are run with AntreaIPAM enabled?

Reply:
ditto